### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by slashplusweb**
Just applying vue js styles according to our style guide and styling the code a bit to make it more readable.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ dist/mix-manifest.json
 dist/swagger-ui.css
 dist/swagger-ui.js
 dist/app.js.LICENSE.txt
+dist/0.js
```
Comment (bambamboole) [inline .gitignore:13]: We should maybe ignore *.js and *.css :-)

File: resources/js/components/Dropdown.vue [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <template>
-    <div v-on-clickaway="away" class="ml-3 relative">
+    <div
+        class="ml-3 relative"
+        v-on-clickaway="away"
+    >
```
Comment (bambamboole) [inline resources/js/components/Dropdown.vue:5]: What is the Rule/Convention how these things will be wrapped? 
Actually for me it is fine how this line was...

File: resources/js/components/Tabs.vue [hunk 3]
```diff
@@ -9,13 +9,14 @@
...
             >
-                <a v-html="tab.header"
-                   :aria-controls="tab.hash"
-                   :aria-selected="tab.isActive"
-                   @click="selectTab(tab.hash, $event)"
-                   :href="tab.hash"
-                   class="tabs-component-tab-a"
-                   role="tab"
+                <a
+                    v-html="tab.header"
```
Comment (bambamboole) [inline resources/js/components/Tabs.vue:13]: I dont like this. There has to be an attribute behind the a this does not look nice to me this way.

File: resources/js/components/fields/ImageField.vue [hunk 4]
```diff
@@ -1,25 +1,30 @@
...
+        :name="field.name" :label="field.label || field.name" :errors="validationErrors" :with-shadow="false">
         <div v-if="value || dataUrl">
             <div class="py-4 flex justify-center">
-                <img :src="imageUrl"
-                     alt
-                     class="rounded w-40 h-40"
-                     :class="{'rounded-full': field.rounded}"
-                     ref="img"
+                <img
+                    :src="imageUrl"
```
Comment (bambamboole) [inline resources/js/components/fields/ImageField.vue:7]: same as anchor tag...

File: resources/js/components/fields/MarkdownField.vue [hunk 5]
```diff
@@ -1,19 +1,24 @@
...
+    <FieldWrapper :name="field.name" :label="field.label" :errors="validationErrors">
         <div class="markdown-toolbar">
             <div class="markdown-modes my-3">
                 <ul class="flex border-b">
                     <li class="-mb-px mr-1">
-                        <button type="button" @click="mode = 'write'"
-                                :class="( mode === 'write' ? 'active bg-white text-indigo-700 ': 'border-transparent ') +
-                                    'py-1 px-2 inline-block border-l border-t border-r rounded-t text-indigo-500 hover:text-blue-800 font-semibold focus:outline-none'">
+                        <button
+                            type="button"
```
Comment (bambamboole) [inline resources/js/components/fields/MarkdownField.vue:8]: same as anchor tag

File: resources/js/components/Dropdown.vue (hunk 2 above)
Comment (slashplusweb) [reply resources/js/components/Dropdown.vue:5]: ya, in this case i'm ok with it :)

normally the conventions say, when you start wrapping one attribute you should wrap all attributes. and you should wrap the attributes whenever its easier to read or if the line is getting to long.

File: resources/js/components/Tabs.vue (hunk 3 above)
Comment (slashplusweb) [reply resources/js/components/Tabs.vue:13]: this way we ensure that on tags that have the same indentation the attributes are for themselves aligned at the same indentation no matter how long the tagname is. this makes absolutely sense in my opinion. i know what you mean, you have to get used to it especially in cases like this, when the tag element is very very short. 

heres an example:

bad:
```html
<mylongtagname attribute1="foo"
               attribute2="bar"
               attribute3="baz"/>
<shorttag attribute1="foo"
          attribute2="bar"
          attribute3="baz"/>
```

good:
```html
<mylongtagname 
    attribute1="foo"
    attribute2="bar"
    attribute3="baz"/>
<shorttag 
    attribute1="foo"
    attribute2="bar"
    attribute3="baz"/>
```

**Pull request merged**